narayana-bom-test/pom.xml

```diff
@@ -17,6 +17,11 @@
   <name>Narayana Bom Test</name>
   <description>Narayana BOM for test dependencies</description>
 
+<!--
+	This Bill of Materials (BOM) is for centralising dependency management across the various modules of Narayana.
+    The BOM is intended for development purposes within Narayana. End users should not use it for integrating Narayana into their projects.
```

Comment:
Why should end users not use it?

Reply:
Thanks for you review @mmusgrov , I removed the last sentence because I agree it has a different meaning than I intended. My point was that the BOM is intended for development purpose, so the previous sentence should be enough.

Reply:
And users should still be okay to use it. If they want to use different versions of a dependency then they use the `<exclusions>` tag.

Reply:
Yes, I agree.